.github/workflows/build-bootc-image.yaml

```
@@ -0,0 +1,152 @@
+          EOF
+
+      - name: Log in to OCI registry
+        run: podman login -u '${{ secrets.QUAY_PUBLISHER_USERNAME }}' -p '${{ secrets.QUAY_PUBLISHER_PASSWORD }}' ${{ inputs.oci-registry }}
+
+      - name: Set env vars
+        run: |
+          echo "OCI_IMAGE=${{ inputs.oci-registry }}/${{ inputs.oci-org }}/${{ inputs.demo }}" >> $GITHUB_ENV
+          echo "OCI_IMAGE_TAG=${GITHUB_SHA::8}-${{ env.ARCH }}" >> $GITHUB_ENV
+
+      - name: Build bootc image
+        run: |
```

actionlint (1.7.4)

property "image" is not defined in object type {runner: string}

(expression)